src/views/infra/network/EditTrafficLabel.vue

```diff
@@ -0,0 +1,235 @@
+        </a-form-item>
+        <a-form-item>
+          <span slot="label">
+            {{ $t('label.ovm3networklabel') }}
+            <a-tooltip :title="apiParams.ovm3networklabel.description">
+              <a-icon type="info-circle" style="color: rgba(0,0,0,.45)" />
+            </a-tooltip>
+          </span>
+          <a-input
+            v-decorator="['ovm3networklabel', {
+              initialValue: trafficResource.ovm3networklabel
+            }]"
+            :placeholder="$t('label.network.label.display.for.blank.value')" />
+        </a-form-item>
+        <div :span="24" class="action-button">
+          <a-button @click="closeAction">{{ $t('label.cancel') }}</a-button>
+          <a-button :loading="loading" type="primary" @click="handleSubmit">{{ $t('label.ok') }}</a-button>
+        </div>
+      </a-form>
+    </a-spin>
+  </div>
+</template>
+
+<script>
+import { api } from '@/api'
+
+export default {
+  name: 'EditTrafficLabel',
+  props: {
+    resource: {
+      type: Object,
+      default: () => {}
+    }
+  },
+  data () {
+    return {
+      loading: false,
+      selectedType: undefined,
+      typeLoading: false,
+      traffictype: null,
+      trafficTypes: [],
+      trafficResource: {}
+    }
+  },
+  beforeCreate () {
+    this.form = this.$form.createForm(this)
+    this.apiConfig = this.$store.getters.apis.updateTrafficType || {}
+    this.apiParams = {}
+    if (this.apiConfig.params) {
+      this.apiConfig.params.forEach(param => {
+        this.apiParams[param.name] = param
+      })
+    }
+  },
+  inject: ['parentFetchData'],
+  mounted () {
+    this.fetchData()
+  },
+  methods: {
+    fetchData () {
+      this.typeLoading = true
+
+      api('listTrafficTypes', { physicalnetworkid: this.resource.id })
+        .then(json => {
+          this.trafficTypes = json.listtraffictypesresponse.traffictype || []
+          this.selectedType = this.trafficTypes[0].id || undefined
+          this.trafficResource = this.trafficTypes[0] || {}
+          this.traffictype = this.trafficTypes[0].traffictype || undefined
+        })
+        .catch(error => {
+          this.$notification.error({
+            message: `${this.$t('label.error')} ${error.response.status}`,
+            description: error.response.data.errorresponse.errortext
+          })
+        })
+        .finally(() => {
+          this.typeLoading = false
+        })
+    },
+    onChangeTrafficType (trafficId) {
+      if (!trafficId) return
+      this.trafficResource = this.trafficTypes.filter(item => item.id === trafficId)[0] || {}
+      this.traffictype = this.trafficResource.traffictype || undefined
+    },
+    handleSubmit (e) {
+      e.preventDefault()
+      this.form.validateFields((err, values) => {
+        if (err) {
+          return
+        }
+        this.loading = true
+        const params = {}
+        for (const key in values) {
+          params[key] = values[key] ? values[key].trim() : undefined
```

Comment:
@utchoang Unable to reset the traffic label. Think removing this check might fix it